### ISSUE TITLE
When editing a question, Advanced Flagging link disappears

### ISSUE THREAD

**Issue by danbopes** [SOBotics/AdvancedFlagging#12]
I've noticed that when I edit a question, and save it, the link to AdvancedFlagging disappears. I need to refresh the page in order to get it back, and it's a little annoying.

See this gif:

=========================

![](https://i.imgur.com/cl3mnW6.gif)

**Comment by rjrudman** [comment]
Will try to look into this this week. Looks like SO reloads the post when submitting an edit, so I'll need to investigate which HTTP request to watch

**Comment by Bhargav-Rao** [comment]
This is actually a known issue and affects a few other userscripts as well. Do share the fix for this, if you find out.

**Comment by rjrudman** [comment]
https://stackoverflow.com/a/10172676/563532

**Comment by rjrudman** [comment]
This is now fixed. Thanks for the report @danbopes !


@Bhargav-Rao After investigating the javascript on SO's dev site, there didn't appear to be any feasible way to monkey patch the code they're using. 
The solution I've used is to watch `$('.answercell.')` and `$('.postcell')` for the 'destroyed' event. See above SO link for explanation.
When we see the event triggered, we parse the new post based on its ID.
